app/_hub/kong-inc/proxy-cache/index.md

```diff
@@ -42,61 +42,71 @@ params:
     - name: response_code
       required:
       default: 200, 301, 404
-      value_in_examples:
+      value_in_examples: [ "200" ]
+      datatype: array integer elements
       description: |
         Upstream response status code considered cacheable.
     - name: request_method
       required:
       default: '`["GET","HEAD"]`'
-      value_in_examples:
+      value_in_examples: ["GET","HEAD"]
+      datatype: array of string elements
       description: |
         Downstream request methods considered cacheable.
     - name: content_type
       required:
       default: '`["text/plain", "application/json"]`'
-      value_in_examples:
+      value_in_examples: ["text/plain", "application/json"]
+      datatype: array of string elements
```

Comment:
In other plugins (for example, key auth), you've used the construction "array of strings". I'm partial to that one - it's shorter, and still clear. Whichever option you go with though, ideally we should keep it consistent.

Reply:
> In other plugins (for example, key auth), you've used the construction "array of strings". I'm partial to that one - it's shorter, and still clear. Whichever option you go with though, ideally we should keep it consistent.

@murillopaula has sated he prefers "elements" I think it depends on the plugin schema but agree we need to be consistent

Reply:
`array of strings` is good, but we're already using the `array of string elements` convention... so my vote is to stay with the latter option for consistency and less work

Reply:
> `array of strings` is good, but we're already using the `array of string elements` convention... so my vote is to stay with the latter option for consistency and less work

Agree that it is more descriptive and preferable. It is already sprinkled all over beyond basic auth plugin.